buildSrc/src/main/groovy/VersionData.groovy

```diff
@@ -93,7 +93,7 @@ class VersionData{
     }
     
     /**
-     * Inpsect versoin - if not starting with 0.0.0 this means it's a release ando
+     * Inpsect version - if not starting with 0.0.0 this means it's a release ando
```

Comment:
What is `ando`?

Reply:
I will fix following typos:

- Inpsect : Inpspect
- versoin : version
- ando: i will check

Remark: This part is not changed inside this commit / PR. So normally not being reviewed here - or ?

Reply:
My assumption was, that you forgot to change the `ando`. 

In general, it might make sense to use automated spellchecking while running the code formatting (see: #771)